src/web/app/pages-logs/logs-page.component.ts

```diff
@@ -29,8 +41,18 @@ interface SearchLogsFormModel {
 interface QueryParams {
```

Comment:
I'm referring to the header comment not displayed in the diff:
```
Query parameters for HTTP request
```
This is way too generic. The query params here is obviously only for one specific endpoint, namely the logs endpoint.

Reply:
ok! will specify that it is for logs endpoint.